docs/DeveloperGuide.md

```diff
@@ -1,29 +1,240 @@
 # Developer Guide
```

Comment:
Overall, there is formatting issue especially the arguments for a few commands do not follow the UML notation